chainer/optimizer.py

```diff
@@ -437,6 +441,16 @@ def update_one_gpu(self, param, state):
         """
         raise NotImplementedError
 
+    def use_cleargrads(self, use=True):
+        """Enables or disables use of :func:`~chainer.Link.cleargrads` in `update`.
+
+        Args:
+            use (bool): If true, this function enables use of `cleargrads`.
+                If false, disables use of `cleargrads`.
```

Comment:
It would be better to note that zerograds is used by default and in the case of `use=False`.